### ISSUE TITLE
dotnet-user-secrets breaks when project uses implicit imports

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/DotNetTools#242]
Repro:
```xml
<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.1" />
    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.0-msbuild2-final" />
  </ItemGroup>
</Project>
```
dotnet-cli 1.0.0-preview3-004215
Run any `dotnet user-secrets` command

Error:
```
debug: Project file path C:\Users\namc\Documents\Visual Studio 2017\Projects\WebApplication1\src\WebApplication2\WebApplication2.csproj.
debug: C:\Program Files\dotnet\sdk\1.0.0-preview4-004215\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.targets(35,3): error MSB4102: The value "" of the "Project" attribute in element <Import> is invalid. Parameter "path" cannot have zero length. [C:\Users\namc\.nuget\packages\microsoft.extensions.secretmanager.tools\1.0.0-msbuild2-final\tools\FindUserSecretsProperty.targets]
fail: Could not load the MSBuild project 'C:\Users\namc\Documents\Visual Studio 2017\Projects\WebApplication1\src\WebApplication2\WebApplication2.csproj'.
```

cref https://github.com/Microsoft/msbuild/issues/1392

**Comment by natemcmaster** [comment]
Caused by https://github.com/Microsoft/msbuild/issues/1451

**Comment by rainersigwald** [comment]
That particular error is actually caused by https://github.com/dotnet/sdk/blob/c82725bc657ad369ecd4e59bf860acf6205027b6/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets#L28-L35. You should be able to work around it by changing the generate-temp-project-with-an-import-of-the-real-project code to ensure that the generated temp file has the same extension as the user project in https://github.com/aspnet/DotNetTools/blob/6decb66d8cc34084a0cb79368d357d5d58cd31da/src/Microsoft.Extensions.SecretManager.Tools/Internal/ProjectIdResolver.cs#L102

**Comment by natemcmaster** [comment]
Fixed in #243
